megamek/src/megamek/common/util/sorter/NaturalOrderComparator.java

```diff
@@ -36,7 +37,8 @@
  * would be sorted into { "Atlas 0", "Atlas 1", "Atlas 2", "Atlas 5", "Atlas 15" }
  * instead of ASCII's { "Atlas 0", "Atlas 1", "Atlas 15", "Atlas 2", "Atlas 5" }
  */
-public class NaturalOrderComparator implements Comparator<String> {
+public class NaturalOrderComparator implements Comparator<String>, Serializable {
+    private static final long serialVersionUID = -5116813198443091269L;
```

Comment:
Why do we need this to be serializable?

Reply:
This is preventative...  From IDEA's Inspection Reasoning: "Reports on subclasses of java.lang.Comparator which are not also Serializable. Objects of java.util.TreeMap or java.util.TreeSet will become non-Serializable if instantiated with such Comparators. This can result in unexpected and difficult-to-diagnose bugs. Since subclasses of java.lang.Comparator are often stateless, simply marking them Serializable is a small cost to avoid such issues."

Reply:
That certainly is a strange behavior.